admin/acceptance.py

```diff
@@ -554,11 +552,15 @@ def configured_cluster_for_nodes(
         dataset_backend_config_file=dataset_backend_config_file
     )
 
-    configuring = perform(
-        make_dispatcher(reactor),
-        configure_cluster(
+    commands = Effect(Func(
```

Comment:
> This is different from how we use Effect elsewhere in the provisioning code.

Effect performers can return effects, which then get run (much like deferreds). I'm presuming at some point, these hooks will be used by people that don't want to learn about effect, so I want to make sure that they get called at the appropriate time (hence the `Func`). We don't need this for our own code, since the functions we call just return an `Effect` instead of having side-effects.

> Do we still generate our docs using Effect? Will this break that?

Yes and no. The docs hook into lower level tasks. Even if they hit this, as you point out, there can be a no-op implementation.

> Are there any meaningful unit tests [...]

I'm not sure what a meaningful unit test would look like. In any case, I don't think it would be too hard to introduce an intent later.